### PR TITLE
Search and sort list book by title and author for user account

### PR DESCRIPTION
**Pull request by VoTaiTri**

### CODE REVIEW

**Comment by khuongnt1688** [comment]
ok rui!

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -1,10 +1,25 @@
 class BooksController < ApplicationController
+  helper_method :sort_column, :sort_direction
+
   def index
-    @books = Book.paginate page: params[:page], per_page: 20
+    @books = Book.search(params[:search], params[:filter] ).order(sort_column + ' ' + sort_direction).paginate page: params[:page], per_page: 10
```
Comment (dieunb) [inline app/controllers/books_controller.rb]:
```
@books = Book.search(params[:search], params[:filter] ).order(sort_column + ' ' + sort_direction)
@books = @books.paginate page: params[:page], per_page: 10

```

File: app/views/books/index.html.erb [hunk 2]
```diff
@@ -9,15 +8,26 @@
     </section>
   </aside>
   <div class="col-md-8">
-    <div class="books">
+    <%= form_tag books_path, :method => 'get', :id => "books_search" do %>
```
Comment (dieunb) [inline app/views/books/index.html.erb]:
```
method: 'get', id: "books_search"
```

File: app/views/books/index.html.erb [hunk 3]
```diff
@@ -9,15 +8,26 @@
...
-        <span class="col-md-6">Author</span>
-      </div>
+        <div class="col-md-1">
+          Search
+        </div>
+        <div class="col-md-6">
+          <%= text_field_tag :search, params[:search] %>
+        </div>
+        <div class="col-md-3">
+          <%= select_tag("filter", options_for_select([['Title', 1], ['Author', 2]],1)) %>
```
Comment (dieunb) [inline app/views/books/index.html.erb]: `options_for_select([['Title', 1], ['Author', 2]], 1`

**Pull request merged**